src/i18n.js

```diff
@@ -1,14 +1,23 @@
 import i18next from 'i18next';
-import { initReactI18next } from 'react-i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
 import HttpApi from 'i18next-http-backend';
+import { initReactI18next } from 'react-i18next';
+
+//fallback defined by ENV disabled because config cant be imported
+//import config from '/config/default';
```

Comment:
you have to do it like this:
```js
import config from "./config";
const defaultLanguage = config.get('settings.defaultLanguage')
```

Reply:
I tried doing this, but i18n does not seem to like this import. Compiling results in this error:
```
.\src\i18n.js
You attempted to import /config which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.
You can either move it inside src/, or add a symlink to it from project's node_modules/.
```
I was not sure if adding a symlink is the right way to go in this case.